### DISCUSSION TITLE
Proposal: a static files directory that gets auto-published

### DISCUSSION THREAD

**Discussion by llimllib** [observablehq/framework#1149] [Ideas]
## The use case:

I'm making a chart and I want to include some team logos, so I stick them in a folder, maybe `/docs/logos`.

Then I make a plot:
```
Plot.plot({marks: [Plot.image(stats, { x: "a", y: "b", src:`docs/logos/${team}.svg`})]})
```

Of course, this doesn't work because the SVGs aren't referenced anywhere that observable knows about, and so the path `/docs/logos` isn't found.

## The proposal:

`/docs/static` or `/static` or something like that ought to be a directory for static assets to live that you can reference unconditionally, and is always carried over into the built site. That way, you can just stick images somewhere in `/docs/static`, reference them in a plot, and no monkey business is required.

## The alternatives:

I know that I could make a `FileAttachement` for each image, or [zip them all up](https://observablehq.com/framework/lib/zip) and pull them out of a zip file.

Both of those seem like unnecessaary work, when all I want to do is reference the images in a graph.

(Have I missed an alternative?)

**Comment by Fil** [comment]
This comes up often and is discussed in https://github.com/observablehq/framework/issues/169.

**Reply by llimllib** [reply]
Thanks, I did look but somehow missed that issue

**Reply by llimllib** [reply]
So that’s just a `wontfix` currently? A reference is required? What is the sensible thing to do in my case with 362 images?

**Reply by Fil** [reply]
I'd say that the answer currently depends on whether these images change on each build. One solution is probably to copy the files over to _files/assets/ after build and before deploy?

> So that’s just a wontfix currently?

I'm pretty sure we'll have to find a solution for this, we're just not there yet. Please keep pushing with example use cases :)
